### DISCUSSION TITLE
Ability to send SMS [through the phone] from another device

### DISCUSSION THREAD

**Discussion by keeganjk** [octoshrimpy/quik#154] [Ideas]
It is slightly inconvenient when working on a desktop computer and then needing to switch to the phone; especially so for any longer messages.

It would be very nice if there was a desktop application, or even just an HTML page that the mobile app would host for me to connect to to send messages. As of yet I have not had an experience with SMS on the computer but this is a great advantage of other instant messaging platforms.

**Comment by lascapi** [comment]
Hi, as workaround you can use https://kdeconnect.kde.org/ :)

**Comment by octoshrimpy** [comment]
I second KDE connect. It is my preferred way of doing this, and if offers other great benefits, like muting audio playback on PC when your phone rings and phone storage mounted over wifi. sadly, it only works within the same local network. for a workaround of that, there would need to be a server setup somewhere to listen to incoming requests and forward them to the phone. Having a server like that running on the phone itself would probably chew through the battery life pretty quick.

**Comment by keeganjk** [comment]
I will have to re-look into KDE Connect but so far I realize it needs to be on the same WiFi network, which will seemingly not work for one network I frequently use. I also have an issue launching the program on one of my machines which has a package management issue which I will need to deal with.

Nevertheless, thank you for the suggestion! This seems like a good approach without complicating the tool itself.

**Reply by yekibud** [reply]
Unfortunately, kde-connect works pretty poorly for me on linux mint.  What do you think about matrix bridges like [android-sms](https://gitlab.com/beeper/android-sms) @octoshrimpy ?  Btw, I couldnt find the #quik-sms matrix room on matrix.org - does that exist/work?

**Reply by octoshrimpy** [reply]
> Unfortunately, kde-connect works pretty poorly for me on linux mint. What do you think about matrix bridges like [android-sms](https://gitlab.com/beeper/android-sms) @octoshrimpy ? Btw, I couldnt find the #quik-sms matrix room on matrix.org - does that exist/work?

Ive thought about matrix bridges for a long time. if you go backwards in time through the history of the readme, you'll see my initial thought was an app that held a lot more protocols in one place, imagine Beeper/bridges but local to device, not to a server. It would probably have to be a plugin or disabled-by-default module, methinks.

try `#quik:matrix.org` as the channel name?

**Reply by yekibud** [reply]
Nope, no dice with #quik, either.  You sure it's public?

**Reply by octoshrimpy** [reply]
> Nope, no dice with #quik, either. You sure it's public?

https://matrix.to/#/#quik:matrix.org

This works for me.

**Comment by eddddddddddddd** [comment]
KDE Connect SMS has always worked horribly for me on Windows.
